### PR TITLE
Add utoipa signer keys

### PR DESCRIPTION
**Pull request by jferrant**
Closes https://github.com/Trust-Machines/stacks-sbtc/issues/420

I recommend checking out the branch and doing the following for some visualzzz:
`cargo build; target/debug/stacks-signer-api swagger`
Open browser and navigate to `http://0.0.0.0:3030/swagger-ui/`

### CODE REVIEW

**Review by sergey-shandar — APPROVED** [review]
@jferrant could you also add a link to the project from the root README.md? Like https://github.com/Trust-Machines/stacks-sbtc#projects

File: stacks-signer-api/README.md [hunk 1]
```diff
@@ -1,37 +1,29 @@
...
+To get started, first, make sure you have Rust installed. You can follow the Rust installation instructions [here](https://www.rust-lang.org/tools/install).
 
-## Installation
+Next, clone this project and navigate to its root directory in your command prompt or terminal.
 
 1. Clone the repository:
 
 ```bash
-git clone git@github.com:Trust-Machines/core-eng.git
+git clone git@github.com:Trust-Machines/stacks-sbtc.git
```
Comment (xoloki) [inline stacks-signer-api/README.md:20]: Not specific to the change, but many users of github who are not devs will not have SSH keys installed, and will instead need to use http(s) links

**Review by xoloki — APPROVED** [review]
One small comment, but overall LGTM

File: stacks-signer-api/README.md (hunk 1 above)
Comment (jferrant) [reply stacks-signer-api/README.md:20]: Good catch!

**Comment by jferrant** [comment]
> @jferrant could you also add a link to the project from the root README.md? Like https://github.com/Trust-Machines/stacks-sbtc#projects

Will do!

**Pull request merged**